src/OrchardCore.Modules/OrchardCore.Media.Azure/Services/MediaBlobContainerTenantEvents.cs

```diff
@@ -10,13 +10,12 @@
 
 namespace OrchardCore.Media.Azure.Services;
 
-public class MediaBlobContainerTenantEvents : ModularTenantEvents
+public sealed class MediaBlobContainerTenantEvents : ModularTenantEvents
 {
     private readonly MediaBlobStorageOptions _options;
     private readonly ShellSettings _shellSettings;
     private readonly ILogger _logger;
-
-    protected readonly IStringLocalizer S;
+    private readonly IStringLocalizer S;
```

Comment:
```suggestion

    internal readonly IStringLocalizer S;
```

Reply:
How this change is useful?

Reply:
because using `S` in a private field goes against the rules we have in the .editorconfig file. I am sure even VS showed you a suggestion to change it. you can't use protected anymore since you sealed the class. The best next option is using internal.

Reply:
Which rules? `internal` doesn't make sense at all

Reply:
private fields should start with `_` otherwise you'll get a warning. look at the code base we do this everywhere the client is sealed. here is a sample https://github.com/OrchardCMS/OrchardCore/blob/b9bbef123cd84f41430e0de1b24076d0ced33a39/src/OrchardCore.Modules/OrchardCore.ContentFields/Drivers/TextFieldDisplayDriver.cs#L15